tap-schema/build-all

```diff
@@ -4,9 +4,21 @@
 # values that control the presentation order of schemas.  This is respected by the Portal Aspect.
 #
 ./build mock
-./build idfprod ../yml/dp02_dc2.yaml ../yml/dp02_obscore.yaml ../yml/dp01_dc2.yaml
-./build idfint ../yml/dp02_dc2.yaml ../yml/dp02_obscore.yaml ../yml/dp01_dc2.yaml
-./build idfdev ../yml/dp02_dc2.yaml ../yml/dp02_obscore.yaml ../yml/dp01_dc2.yaml
-./build idfsso ../yml/dp03.yaml
-./build usdf-dev ../yml/dp02_dc2.yaml ../yml/dp02_obscore.yaml ../yml/oga_live_obscore.yaml
-./build usdf-prod ../yml/dp02_dc2.yaml ../yml/dp02_obscore.yaml
+
+./build tap-schema-idfprod-tap ../yml/dp02_dc2.yaml ../yml/dp01_dc2.yaml
+./build tap-schema-idfint-tap ../yml/dp02_dc2.yaml ../yml/dp01_dc2.yaml
+./build tap-schema-idfdev-tap ../yml/dp02_dc2.yaml ../yml/dp01_dc2.yaml
+./build tap-schema-usdf-dev-tap ../yml/dp02_dc2.yaml
+./build tap-schema-usdf-prod-tap ../yml/dp02_dc2.yaml
+
+./build tap-schema-idfprod-obstap ../yml/dp02_obscore.yaml
+./build tap-schema-idfint-obstap ../yml/dp02_obscore.yaml
+./build tap-schema-idfdev-obstap ../yml/dp02_obscore.yaml
+./build tap-schema-usdf-dev-obstap ../yml/dp02_obscore.yaml ../yml/oga_live_obscore.yaml
+./build tap-schema-usdf-prod-obstap ../yml/dp02_obscore.yaml
```

Comment:
I believe `dp02_obscore.yaml` is served by qserv everywhere, and the only separate obstap is live_obscore. So I think this should be:
```suggestion
./build tap-schema-idfprod-tap ../yml/dp02_dc2.yaml ../yml/dp02_obscore.yaml ../yml/dp01_dc2.yaml
./build tap-schema-idfint-tap ../yml/dp02_dc2.yaml ../yml/dp02_obscore.yaml ../yml/dp01_dc2.yaml
./build tap-schema-idfdev-tap ../yml/dp02_dc2.yaml ../yml/dp02_obscore.yaml ../yml/dp01_dc2.yaml
./build tap-schema-usdf-dev-tap ../yml/dp02_dc2.yaml ../yml/dp02_obscore.yaml
./build tap-schema-usdf-prod-tap ../yml/dp02_dc2.yaml ../yml/dp02_obscore.yaml

./build tap-schema-idfprod-obstap ../yml/oga_live_obscore.yaml
./build tap-schema-idfint-obstap ../yml/oga_live_obscore.yaml
./build tap-schema-idfdev-obstap ../yml/oga_live_obscore.yaml
./build tap-schema-usdf-dev-obstap ../yml/oga_live_obscore.yaml
./build tap-schema-usdf-prod-obstap ../yml/oga_live_obscore.yaml
```

There's really no meaningful obstap at IDF as I understand it so I don't think those schemas in the second block work at the IDF, but including them in case you want them for completeness.

Reply:
Yeah I think you're right about not meaningful obstap at IDF - I was using it to test.  But first I'd like to get the data here, then I'll cut it down to the right environments and do that in another commit.  I think some of these were cross connected where the database was actually talking to usdf but I may be out of my mind on that one.